src/components/Auth/ResetPassword.tsx

```diff
@@ -69,40 +71,37 @@ const ResetPassword = (props: ResetPasswordProps) => {
     }
     return form;
   };
+  const { mutate: resetPassword } = useMutation({
+    mutationFn: mutate(routes.resetPassword),
+    onSuccess: (_data) => {
```

Comment:
if you don't need it, why take it in the first place?

```suggestion
    onSuccess: () => {
```